### PR TITLE
Test suite for `rm` CLI command

### PR DESCRIPTION
**Pull request by hculea**
Hereby, a test suite achieving 82.9% statement coverage for the 'secrethub rm' command, testing the behaviour of this command when called on directories, secrets, and secret versions.

### CODE REVIEW

File: internals/secrethub/rm_test.go [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+		err                error
+		promptError        error
+		deleteSecretError  error
+		deleteVersionError error
+		deleteDirError     error
+		newClientError     error
+		getTreeError       error
+		getSecretError     error
+	}{
+		"success-force-dir": {
```
Comment (Marton6) [inline internals/secrethub/rm_test.go:35]: You can also have spaces in test case names, they will be replaced by underscores by `go test`. For the sake of consistency, I think we should stick to all lowercase, space separated test case names in code.

File: internals/secrethub/rm_test.go [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+	var testErr = errors.New("test")
+
+	cases := map[string]struct {
+		cmd                RmCommand
+		in                 string
+		argPath            api.Path
+		promptOut          string
+		out                string
+		err                error
+		promptError        error
```
Comment (Marton6) [inline internals/secrethub/rm_test.go:27]: For consistency, all errors should either be named `err` or have the `Err` suffix.

File: internals/secrethub/rm_test.go [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+				recursive: true,
+				path:      "namespace/repo/dir",
+			},
+			argPath: "namespace/repo/dir",
+			promptOut: "[WARNING] This action cannot be undone. This will permanently remove the namespace/repo/dir directory and all the directories and secrets it contains. " +
+				"Please type in the name of the directory to confirm: ",
+			in:  "namespace/repo/dir",
+			out: "Removal complete! The directory namespace/repo/dir has been permanently removed.\n",
+		},
+
```
Comment (Marton6) [inline internals/secrethub/rm_test.go:56]: This and all other blank lines between test cases should be removed.

File: internals/secrethub/rm_test.go (hunk 1 above)
Comment (hculea) [reply internals/secrethub/rm_test.go]: This should now be solved! :)

File: internals/secrethub/rm_test.go (hunk 2 above)
Comment (hculea) [reply internals/secrethub/rm_test.go]: This should now be solved! :)

File: internals/secrethub/rm_test.go (hunk 3 above)
Comment (hculea) [reply internals/secrethub/rm_test.go]: This should now be solved! :)

File: internals/secrethub/rm_test.go [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+		"success non force secret version": {
+			cmd: RmCommand{
+				force: false,
+				path:  "namespace/repo/secret:latest",
+			},
+			argPath: "namespace/repo/secret:latest",
+			promptOut: "[WARNING] This action cannot be undone. This will permanently remove the namespace/repo/secret:latest secret version. " +
+				"Please type in the name of the secret and the version (<name>:<version>) to confirm: ",
+			in:  "namespace/repo/secret:latest",
+			out: "Removal complete! The secret version namespace/repo/secret:latest has been permanently removed.\n",
```
Comment (Marton6) [inline internals/secrethub/rm_test.go:138]: We could also make a constant for the warning and the removal complete messages.

**Pull request merged**